front/app/containers/Admin/users/ChangeSeatModal.tsx

```diff
@@ -149,6 +149,7 @@ const ChangeSeatModal = ({
           </Box>
           <Box display="flex" width="100%" alignItems="center">
             <Button
+              autoFocus={true}
```

Comment:
Minor: Since we are passing `true`, we can just do
```suggestion
              autoFocus
```